### PR TITLE
[DOC-8435][DOC-8546] Document how to get the connection string for MR Serverless with PrivateLink, correct VPC Peering error

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-8435] Document how to get the connection string for MR Serverless with PrivateLink
[DOC-8546] Correct VPC Peering docs to change the desired status from Active to Available

- Also improve the Dedicated doc in a similar way.

Preview: 
[src/current/cockroachcloud/connect-to-a-serverless-cluster.md](https://deploy-preview-17570--cockroachdb-docs.netlify.app/docs/cockroachcloud/connect-to-a-serverless-cluster.html)
[src/current/cockroachcloud/connect-to-your-cluster.md](https://deploy-preview-17570--cockroachdb-docs.netlify.app/docs/cockroachcloud/connect-to-your-cluster.html)

### CODE REVIEW

File: src/current/cockroachcloud/connect-to-your-cluster.md [hunk 1]
```diff
@@ -113,10 +113,16 @@ To connect to your cluster with the [built-in SQL client](../{{site.current_clou
...
     {% include cockroachcloud/download-the-cert.md %}
 
+1. If you [established a private connection using VPC Peering or AWS PrivateLink](#establish-gcp-vpc-peering-or-aws-privatelink), change **Connection Type** from **public connection** to **private connection** to connect privately.
+
 1. Copy the [`cockroach sql`](../{{site.current_cloud_version}}/cockroach-sql.html) command and connection string provided in the Console, which will be used in the next step (and to connect to your cluster in the future):
 
     {% include cockroachcloud/sql-connection-string.md %}
 
+    {{site.data.alerts.callout_info}}
+
```
Comment (gemma-shay) [inline src/current/cockroachcloud/connect-to-your-cluster.md:123]: empty callout?

File: src/current/cockroachcloud/connect-to-your-cluster.md [hunk 2]
```diff
@@ -113,10 +113,16 @@ To connect to your cluster with the [built-in SQL client](../{{site.current_clou
...
+    {{site.data.alerts.end}}
+
```
Comment (mdlinville) [inline src/current/cockroachcloud/connect-to-your-cluster.md:125]:
```suggestion
```

**Review by gemma-shay — APPROVED** [review]

File: src/current/cockroachcloud/connect-to-a-serverless-cluster.md [hunk 3]
```diff
@@ -68,6 +68,7 @@ Private connectivity is not available for {{ site.data.products.serverless }} cl
   <section class="filter-content" markdown="1" data-scope="connection-string">
 
 1. In the **Download CA Cert** section of the dialog, select your operating system, and use the command provided to download the CA certificate to the default PostgreSQL certificate directory on your machine.
+1. If you [established a private connection using AWS PrivateLink](#establish-aws-privatelink), change **Connection Type** from **public connection** to **private connection** to connect privately.
```
[the remark this answers is not in the document]
Comment (mdlinville) [reply src/current/cockroachcloud/connect-to-a-serverless-cluster.md:71]: I double-checked and it is actually **Connection type**, **Public connection**, and **Private connection**.

**Review by abhinavg6 — APPROVED** [review]
LGTM

**Review by gemma-shay — APPROVED** [review]

**Pull request merged**